src/editor/button_widget.cpp

```diff
@@ -43,12 +43,10 @@ ButtonWidget::draw(DrawingContext& context)
   }
 
   if (m_grab) {
-    context.color().draw_filled_rect(m_rect, Color(g_config->editorcolor.red + 0.4f, g_config->editorcolor.green + 0.4f,
-      g_config->editorcolor.blue + 0.4f, g_config->editorcolor.alpha + 0.1f), 4.0f,
+    context.color().draw_filled_rect(m_rect, Color(g_config->editorgrabcolor), 4.0f,
```

Comment:
At a few places you cann the `Color(Color&)` copy constructor; is there a reason? Wouldn't just `g_config->editorgrabcolor` work?